vint.go

```diff
@@ -60,3 +60,46 @@ func mustReadByte(r io.Reader) (byte, error) {
 
 	return b, err
 }
+
+func WriteVInt(w io.Writer, n int64) (error) {
```

Comment:
It might be useful to return the number of bytes written (I forgot that for `ReadVInt`)